cvat/apps/opencv/views.py

```diff
@@ -0,0 +1,7 @@
+import os
+from django.conf import settings
+from sendfile import sendfile
+
+def OpenCVLibrary(request):
+    path = os.path.realpath(os.path.join(settings.STATIC_ROOT, 'opencv', 'js', 'opencv.js'))
+    return sendfile(request, path)
```

Comment:
I like the solution with a separate app for OpenCV.js. Great job!

Reply:
Probably you want to include version information into the filename. For example, opencv-X.Y.Z.js. The file will be accessed using opencv.js but in repo we will see which version we have.

Reply:
Please note that related OpenCV code is not released yet (it is missing in 4.5.1, 4.5.2 is not released yet)

Reply:
Better to have something in any case. Probably 4.5.1 is better than nothing.

Reply:
Alternative is using package/PR creation date (20210120 or without day 202101):
`opencv-cvat-<YYYYmmdd>.js`

Reply:
Done